src/com/magento/idea/magento2plugin/actions/generation/dialog/validator/CreateAPluginDialogValidator.java

```diff
@@ -1,259 +0,0 @@
-        if (pluginName.length() == 0) {
-            final String errorMessage = validatorBundle.message(
-                    NOT_EMPTY,
-                    "Plugin Name"
-            );
-            JOptionPane.showMessageDialog(
-                    null,
-                    errorMessage,
-                    errorTitle,
-                    JOptionPane.ERROR_MESSAGE
-            );
-
-            return false;
-        }
-
-        if (!pluginName.matches(RegExUtil.IDENTIFIER)) {
-            final String errorMessage = validatorBundle.message(
-                    "validator.identifier",
-                    "Plugin Name"
-            );
-            JOptionPane.showMessageDialog(
-                    null,
-                    errorMessage,
-                    errorTitle,
-                    JOptionPane.ERROR_MESSAGE
-            );
-
-            return false;
-        }
-
-        final String sortOrder = dialog.getPluginSortOrder();
-        if (sortOrder.length() == 0) {
-            final String errorMessage = validatorBundle.message(
-                    NOT_EMPTY,
-                    "Sort Order"
-            );
-            JOptionPane.showMessageDialog(
-                    null,
-                    errorMessage,
-                    errorTitle,
-                    JOptionPane.ERROR_MESSAGE
-            );
-
-            return false;
-        }
-
-        if (!sortOrder.matches(RegExUtil.NUMERIC)) {
-            final String errorMessage = validatorBundle.message(
-                    "validator.onlyNumbers",
-                    "Sort Order"
-            );
-            JOptionPane.showMessageDialog(
-                    null,
-                    errorMessage,
-                    errorTitle,
-                    JOptionPane.ERROR_MESSAGE
-            );
-
-            return false;
-        }
-
-        final String pluginModule = dialog.getPluginModule();
-        if (pluginModule.length() == 0) {
-            final String errorMessage = validatorBundle.message(
-                    NOT_EMPTY,
-                    "Plugin Module"
-            );
-            JOptionPane.showMessageDialog(
-                    null,
-                    errorMessage,
-                    errorTitle,
-                    JOptionPane.ERROR_MESSAGE
-            );
-
-            return false;
-        }
-
-        final List<String> allModulesList = ModuleIndex
-                .getInstance(project).getEditableModuleNames();
-        if (!allModulesList.contains(pluginModule)) {
-            final String errorMessage = validatorBundle.message(
-                    "validator.module.noSuchModule",
-                    pluginModule
-            );
-            JOptionPane.showMessageDialog(
-                    null,
-                    errorMessage,
-                    errorTitle,
-                    JOptionPane.ERROR_MESSAGE
-            );
-
-            return false;
-        }
```

Comment:
This isn't implemented and will be done in a separate PR as per @VitaliyBoyko.

Reply:
Could you please create an Issue and reference it to this PR. TY

Reply:
Hi @VitaliyBoyko, I've created an issue for the above: https://github.com/magento/magento2-phpstorm-plugin/issues/332. Thanks.

Reply:
Hi @VitaliyBoyko, I've closed the above issue and suggested a better aproach in this comment: https://github.com/magento/magento2-phpstorm-plugin/pull/326#issuecomment-714742389.